plugins/meta/flannel/flannel_windows.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2018 CNI authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This is a "meta-plugin". It reads in its own netconf, combines it with
+// the data from flannel generated subnet file and then invokes a plugin
+// like bridge or ipvlan to do the real work.
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/containernetworking/cni/pkg/invoke"
```

Comment:
Looks like you need to run gofmt on this file, it's causing this build to fail https://travis-ci.org/containernetworking/plugins/builds/428214643

Reply:
@benmoss, thx